src/rgw/rgw_common.h

```diff
@@ -757,6 +757,11 @@ enum RGWBucketFlags {
   BUCKET_VERSIONS_SUSPENDED = 0x4,
 };
 
+enum RGWBucketIndexType {
+  RGWBIType_Normal = 0,
```

Comment:
@oritwas How good to take 'RGWBIType_NORMAL' instead of 'RGWBIType_Normal'?
https://google.github.io/styleguide/cppguide.html#Enumerator_Names